docs/migrating_from_1x.md

```diff
@@ -0,0 +1,26 @@
+# Migrating Simulations from Shadow 1.x
+
+Shadow 2.0 has changed the formatting of its configuration and topology files. The configuration format has changed from XML to YAML, and the topology format has changed from GraphML to GML. Various options have been added, renamed, and removed. Shadow includes convenience scripts to convert these files to their new formats. These scripts are intended to aid in converting to the new formats, and you should manually compare the original file to the new converted file to make sure it includes all of the options you expect.
+
+When encountering a tag/attribute that is not supported by the new format, these scripts will either:
+
+1. Copy it to the new file anyways. When attempting to use this new file with Shadow, Shadow will raise an error for this unexpected field.
+2. Ignore it and output a warning.
```

Comment:
What determines which action is taken? Is `2.` only for "whitelisted" attributes that we've deemed safe to drop? Or is it controlled by a command-line argument to the conversion script? Or...?